gitlab/client.py

```diff
@@ -529,6 +531,7 @@ def enable_debug() -> None:
         requests_log = logging.getLogger("requests.packages.urllib3")
         requests_log.setLevel(logging.DEBUG)
         requests_log.propagate = True
+        LOG.debug("Enabled debug mode for python-gitlab")
```

Comment:
Is this needed here? Thinking in general: I assume this currently has the effect that `enable_debug()` will start logging requests & http.client but not python-gitlab (because of the null handler) so this is a no-op unless they also manually set up their own logging. I think that libraries with explicit `debug=True` flags/`enable_debug()`/`add_logger()` etc methods, will set up some kind of default stream handler in those, for the current logger so this is actually displayed. E.g.

https://github.com/pallets/flask/blob/2f0c62f5e6e290843f03c1fa70817c7a3c7fd661/src/flask/logging.py#L45-L74
https://github.com/encode/httpx/blob/2434e650eeeedd0652ab73c02bcb09417edc7914/httpx/_utils.py#L234-L258

Reply:
Well the goal was to enable basic logging and use it. So if they do setup logging in the calling applications they will see this log message.

The logging documentation states as a library we should use the NullHandler and it is the responsibility of the caller to enable logging.

But I can also see if the explicitly call `enable_debug()` that we could do something to turn on logging output if they haven't set it up in the caller. But I sort of feel that is outside the scope of this "bare-minimum logging support"